### PR TITLE
Fixed missing URL escaping issue

### PR DESCRIPTION
**Pull request by girishpanchal30**
Closes Codeinwp/ppom-pro#202

### CODE REVIEW

**Review by mskapusuz — CHANGES_REQUESTED** [review]
for the redirects, we should use [esc_url_raw](https://developer.wordpress.org/reference/functions/esc_url_raw/) instead of `esc_url`

we should use `esc_url` for needs to be displayed

**Comment by girishpanchal30** [comment]
@mskapusuz Thank you for review and suggestion, applied mentioned changes in the latest commit.

File: inc/admin.php [hunk 1]
```diff
@@ -299,7 +299,7 @@ function ppom_admin_save_form_meta() {
 			'message'        => __( 'Form added successfully', 'woocommerce-product-addon' ),
 			'status'         => 'success',
 			'productmeta_id' => $ppom_id,
-			'redirect_to'    => $redirect_to,
+			'redirect_to'    => esc_url( $redirect_to ),
 		);
```
Comment (mskapusuz) [inline inc/admin.php:303]: can you please use `esc_url_raw` here and for other `redirect_to` params in AJAX responses. JS uses that URL to redirect the page.

That's valid for all `redirect_to` params in the AJAX responses, as I see some of them are missing by esc.

Screen Shot 2023-05-03 at 10 02 02

**Review by mskapusuz — CHANGES_REQUESTED** [review]
@girishpanchal30 thank you for your fixes, I've left one thing, other than that it looks good 🚀

File: inc/admin.php (hunk 1 above)
Comment (mskapusuz) [reply inc/admin.php:303]: P.S.: as I see `js/admin/ppom-admin.js` handles the redirection with the AJAX response.

Comment (girishpanchal30) [reply inc/admin.php]: @mskapusuz Fixed, Thank you!

**Review by mskapusuz — APPROVED** [review]

**Comment by irinelenache** [comment]
@mskapusuz Didn't manage to replicate the issue. Can you check if the vulnerability is fixed and move the issue to **Ready to merge**?

**Comment by mskapusuz** [comment]
> @mskapusuz Didn't manage to replicate the issue. Can you check if the vulnerability is fixed and move the issue to **Ready to merge**?

@irinelenache of course, but it's ok in terms of the regression testing, right? Thank you. 🚀

**Comment by irinelenache** [comment]
@mskapusuz Managed to replicate it and indeed the issue is fixed, didn't find any other regressions so i'll move this to Ready to merge 🚀

**Pull request merged**